common/db.go

```diff
@@ -379,9 +390,29 @@ func (db *DB) InsertUnbondingLock(id *big.Int, delegator ethcommon.Address, amou
 	return nil
 }
 
+// DeleteUnbondingLock deletes an unbonding lock from the DB with the given ID and delegator address. This method will
+// return nil for non-existent unbonding locks
+func (db *DB) DeleteUnbondingLock(id *big.Int, delegator ethcommon.Address) error {
+	glog.V(DEBUG).Infof("db: Deleting unbonding lock %v for delegator %v", id, delegator.Hex())
+	_, err := db.deleteUnbondingLock.Exec(id.Int64(), delegator.Hex())
+	if err != nil {
+		glog.Errorf("db: Error deleting unbonding lock %v for delegator %v: %v", id, delegator.Hex(), err)
+		return err
+	}
+	return nil
+}
+
+// UseUnbondingLock sets an unbonding lock in the DB as used by setting the lock's used block. If usedBlock is nil
```

Comment:
Perhaps it's better to start a the new sentence on a new line 

```
// UseUnbondingLock sets an unbonding lock in the DB as used by setting the lock's used block.
// If usedBlock is nil this method will set the lock's used block to NULL
```

Reply:
Fixed in da84f63